docs/content/en/docs/_index.md

```diff
@@ -41,8 +41,6 @@ SOARCA currently supports the following transport mechanisms:
 {{% card header="SSH - Native" %}}
 [![Ssh](/SOARCA/images/logos-external/ssh.svg)](/docs/soarca-extensions/native-capabilities/#ssh-capability)
 {{% /card %}}
-{{< /cardpane >}}
```

Comment:
One of the many errors in the Hugo docs; I could not get Hugo to build with any recent (> Jan 2024) Hugo version.